### PR TITLE
[DNM] Update Performance Result

### PR DESCRIPTION
**Pull request by weiting-chen**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,18 +73,35 @@ Once your env being successfully deployed, please refer to [Gluten Usage](./docs
...
 ### Build Gluten with ClickHouse backend
 
 ![logo](./docs/image/ClickHouse/logo.png)
 
 If you would like to build Gluten with **ClickHouse** backend, please follow the steps in [Build with ClickHouse Backend](./docs/ClickHouse.md). ClickHouse backend is devleoped by [Kyligence](https://kyligence.io/), please visit https://github.com/Kyligence/ClickHouse for more infomation.
 
+# Performance
+
+We use Decision Support Benchmark1(TPC-H Like) to evaluate the performance for Gluten project.
+* Decision Support Benchmark1 is a query set modified from [TPC-H benchmark](http://tpc.org/tpch/default5.asp). Because some features are not fully supported, there are some changes during the testing. We change Decimal to Double and Date to String and use DWRF file format for Velox testing & MergeTree file format for Clickhouse testing compared to Parquet file format as baseline.
```
[the remark this answers is not in the document]
Comment (weiting-chen) [reply README.md]: Done

**Comment by FelixYBW** [comment]
@zzcclp, Can you review and update Clickhouse backend?

File: README.md [hunk 2]
```diff
@@ -83,6 +83,23 @@ If you would like to build Gluten with **Arrow** backend, please follow the step
...
 
+# Performance
+
+We use Decision Support Benchmark1(TPC-H Like) to evaluate the performance for Gluten project.
+* Decision Support Benchmark1 is a query set modified from [TPC-H benchmark](http://tpc.org/tpch/default5.asp). Because some features are not fully supported, there are some changes during the testing. Firstly we change column data type like Decimal to Double and Date to String. Secondly we use [DWRF](https://github.com/oap-project/gluten/tree/main/backends-velox/workload/tpch) file format for Velox testing & MergeTree file format for Clickhouse testing compared to Parquet file format as baseline. Thirdly we modify the SQLs to use double and string data type for both Gluten and baseline. backends-velox/workload/tpch has the script to reproduce the performance number for Velox backend.
+
+The testing environment is using single node with 1TB datasize. The result shows a up to 3.48X speedup in Decision Support Benchmark1 with Gluten and Velox backend.
```
Comment (zzcclp) [inline README.md:91]: Modify: `up to 3.63X speedup` for velox backend

File: README.md [hunk 3]
```diff
@@ -83,6 +83,23 @@ If you would like to build Gluten with **Arrow** backend, please follow the step
...
+![Performance](./docs/image/velox_decision_support_bench1_10query_performance.png)
+
+The testing environment is using a 8-nodes AWS cluster with 1TB datasize. The result shows a up to 3.63X speedup in Decision Support Benchmark1 with Gluten and Clickhouse backend.
```
Comment (zzcclp) [inline README.md:94]: Modify: `up to 3.48X speedup` for clickhouse backend

**Review by FelixYBW — APPROVED** [review]

**Pull request merged**